ignite-generator/app/index.js

```diff
@@ -109,7 +75,7 @@ var AppGenerator = function (_NamedBase) {
       this.spawnCommandSync('react-native', ['init', this.name]);
 
       // Grab latest RNBase into templates folder
-      _shelljs2.default.exec('git clone git@github.com:infinitered/react_native_base.git ' + this.templateFolder);
+      Shell.exec('git clone git@github.com:infinitered/react_native_base.git ' + this.templateFolder);
```

Comment:
at some point we can.  For now all git paths will autoforward since the repo was moved.  Feel free to PR the change.